test/models/user_test.rb

```diff
@@ -18,4 +18,11 @@ class UserTest < ActiveSupport::TestCase
 
     refute user.valid?
   end
+
+  test 'a user has an email' do
+    user = User.new(username: 'Matt', password: 'password', email: 'test@email.com')
```

Comment:
Line is too long. [84/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.